site/_posts/2021-03-16-securely-implement-oauth-vuejs.md

```diff
@@ -586,7 +623,7 @@ The output of the above is the same as when we have hardcoded the `email` value
 
 If we comment out `email` in `server/routes/user.js`, we will see the "You are not logged in" message in our application. We can change the email in `server/routes/user.js` and see the corresponding DOM changes as well:
 
-```
+```shell
```

Comment:
This is a javascript fragment, right? Not shell?

Also, where we have a fragment, I use

`// ...`

To indicate there is more above or below the code.